shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -38,21 +38,26 @@ - (void)sendKeyEvent:(const FlutterKeyEvent&)event
 /// Sometimes we have to use a custom mock to avoid retain cycles in OCMock.
 /// Used for testing low memory notification.
 @interface FlutterEnginePartialMock : FlutterEngine
+
 @property(nonatomic, strong) FlutterBasicMessageChannel* lifecycleChannel;
 @property(nonatomic, strong) FlutterBasicMessageChannel* keyEventChannel;
 @property(nonatomic, weak) FlutterViewController* viewController;
 @property(nonatomic, strong) FlutterTextInputPlugin* textInputPlugin;
 @property(nonatomic, assign) BOOL didCallNotifyLowMemory;
+
 - (FlutterTextInputPlugin*)textInputPlugin;
+
 - (void)sendKeyEvent:(const FlutterKeyEvent&)event
             callback:(nullable FlutterKeyEventCallback)callback
             userData:(nullable void*)userData;
 @end
 
 @implementation FlutterEnginePartialMock
-@synthesize viewController;
+
+// Synthesize properties declared readonly in FlutterEngine.
 @synthesize lifecycleChannel;
 @synthesize keyEventChannel;
+@synthesize viewController;
 @synthesize textInputPlugin;
```

Comment:
No changes, just reordered the existing declarations to match the property declarations above.